app/validators/claim/advocate_claim_validator.rb

```diff
@@ -72,11 +75,21 @@ def validate_defendant_uplifts
 
   # we add one because uplift quantities reflect the number of "additional" defendants
   def defendant_uplifts_greater_than?(no_of_defendants)
-    @record
-      .fees
-      .where.not(id: @record.misc_fees.select(&:marked_for_destruction?).map(&:id))
-      .defendant_uplift_sums
-      .values
-      .map(&:to_i).any? { |sum| sum + 1 > no_of_defendants }
+    defendant_uplifts_counts.map(&:to_i).any? { |sum| sum + 1 > no_of_defendants }
+  end
+
+  def defendant_uplifts
+    (@record.basic_fees + @record.fixed_fees + @record.misc_fees).select do |fee|
+      !fee.marked_for_destruction? &&
+        fee.fee_type &&
+        Fee::BaseFeeType.defendant_uplift_unique_codes.include?(fee.fee_type.unique_code)
```

Comment:
There is a `defendant_uplift?` method that could be used here - see `DefendantUpliftable` module